firebase_admin/__init__.py

```diff
@@ -242,7 +243,7 @@ def _lookup_project_id(cls, credential, options):
             except AttributeError:
                 pass
         if not pid:
```

Comment:
hmm. in hindsight pid seems like an unfortunate name for project id.  Can we change that?

Reply:
Done